### PR TITLE
Fixes #530 AP and CP abbreviations replaced in table

### LINKED ISSUE #530: Docs: Feedback for Distributed Data Structures
Hi, I have some feedback about [this page](https://github.com/hazelcast/hz-docs/edit/main/docs/modules/data-structures/pages/distributed-data-structures.adoc)

The terms AP & CP is not abbreviated to provide the reader a quick understanding. From what I could presume
AP- Availability and Partition tolerance 
CP - Consistency and Partition tolerance.
I was able to locate this analogy from other pages or elsewhere in the internet, Can this be improved?

### PR DESCRIPTION
**Pull request by angelasimms**
Closes issue #530.

### CODE REVIEW

File: docs/modules/data-structures/pages/distributed-data-structures.adoc [hunk 1]
```diff
@@ -8,119 +8,120 @@ All of these structures are available in Java, .NET, C++, Node.js, Python, and G
 
 [cols="20%a,40%a,20%a,20%a"]
 |===
-|Data structure |Description|Partitioned/Non-partitioned|AP/CP
+|Data structure |Description|Partitioned/Non-partitioned|Tolerance
```
Comment (JakeSCahill) [inline docs/modules/data-structures/pages/distributed-data-structures.adoc:11]: It's really about availability, consistency, and partition tolerance (CAP). CAP theorem states that a system can deliver two of the three aspects at one time.

File: docs/modules/data-structures/pages/distributed-data-structures.adoc [hunk 2]
```diff
@@ -8,119 +8,120 @@ All of these structures are available in Java, .NET, C++, Node.js, Python, and G
...
 
 |xref:map.adoc[Map]
 |Key-value pairs that are partitioned across a cluster. Maps offer a wide range of features such as SQL queries, WAN replication, and Near Cache.
-|Partitioned
-|AP
+|<<partitioned,Partitioned>>
+|<<ap-data,Availability and partition>>
```
Comment (JakeSCahill) [inline docs/modules/data-structures/pages/distributed-data-structures.adoc:16]:
```suggestion
|<<ap-data,Availability and partition tolerance>>
```

File: docs/modules/data-structures/pages/distributed-data-structures.adoc [hunk 3]
```diff
@@ -8,119 +8,120 @@ All of these structures are available in Java, .NET, C++, Node.js, Python, and G
...
 
 |xref:fencedlock.adoc[Fenced Lock]
 |A lock that provides exclusive access to a shared resource; only
 one thread at a time can acquire the lock and all access to the shared resource requires
 that the lock be acquired first.
-|Non-Partitoned
-|CP
+|<<partitioned,Non-partitioned>>
+|<<cp-data,Consistency and partition>>
```
Comment (JakeSCahill) [inline docs/modules/data-structures/pages/distributed-data-structures.adoc:79]:
```suggestion
|<<cp-data,Consistency and partition tolerance>>
```

**Review by JakeSCahill — CHANGES_REQUESTED** [review]
Thanks! Left a comment and some suggestions.

File: docs/modules/data-structures/pages/distributed-data-structures.adoc (hunk 1 above)
Comment (angelasimms) [reply docs/modules/data-structures/pages/distributed-data-structures.adoc]: I've linked the column heading to the relevant section in the overview that gives a fuller explanation. WDYT?

**Review by JakeSCahill — APPROVED** [review]

**Pull request merged**